### PR TITLE
BUGZ-837: Fix Launcher crash (mac)

### PR DESCRIPTION
**Pull request by danteruiz**
Fix launcher crash (mac) - ticket https://highfidelity.atlassian.net/browse/BUGZ-837

### CODE REVIEW

File: launchers/darwin/src/LatestBuildRequest.m [hunk 1]
```diff
@@ -6,12 +6,18 @@
...
+    @try {
+        NSString* interfaceAppPath = [[Launcher.sharedLauncher getAppPath] stringByAppendingString:@"interface.app"];
+        NSError * error = nil;
+        Interface * interface = [[Interface alloc] initWith:interfaceAppPath];
+        currentVersion = [interface getVersion:&error];
+        if (currentVersion == 0 && error != nil) {
+            NSLog(@"can't get version from interface, falling back to settings: %@", error);
+            currentVersion = [Settings.sharedSettings latestBuildVersion];
+        }
+    } @catch (NSException *exception) {
```
Comment (MattHardcastle) [inline launchers/darwin/src/LatestBuildRequest.m]: `NSLog(@"an exception was thrown: %@", exception);` Something like this should work. **untested**

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20146/HighFidelity-Beta-PR15840-1e38e7b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20146/HighFidelity-Quest-Beta-PR15840-1e38e7b.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20146/HighFidelity-Beta-PR15840-1e38e7b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20146/HighFidelity-Beta-Interface-PR15840-1e38e7b.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20146/HighFidelity-Beta-PR15840-1e38e7b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20146/HighFidelity-Beta-Interface-PR15840-1e38e7b.dmg

**Pull request merged**